src/main/java/frc/robot/subsystems/intake/IntakeOpenerSS.java

```diff
@@ -0,0 +1,58 @@
+package frc.robot.subsystems.intake;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import frc.robot.components.TrigonTalonSRX;
+import frc.robot.constants.RobotConstants.IntakeOpenerConstants;
+import frc.robot.subsystems.OverridableSubsystem;
+import frc.robot.utilities.EncoderConversions;
+
+public class IntakeOpenerSS extends OverridableSubsystem {
+    private final TrigonTalonSRX motor;
+
+    public IntakeOpenerSS() {
+        motor = IntakeOpenerConstants.MOTOR;
+    }
+
+    /**
+     * @param power to be applied to the motors
+     */
+    public void setPower(double power){
+        motor.set(power);
+    }
+    @Override
+    public void overriddenMove(double power) {
+        setPower(power);
+    }
```

Comment:
remove

Reply:
why to remove it. lavi and natan said to keep.